exercises/concept/resistor-color/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Debug, PartialEq)]
+pub enum ResistorColor {
+    Black,
+    Blue,
+    Brown,
+    Green,
+    Grey,
+    Orange,
+    Red,
+    Violet,
+    White,
+    Yellow,
+}
+
+pub fn color_to_value(_color: ResistorColor) -> usize {
+    unimplemented!("convert a color into a numerical representation")
+}
+
+pub fn value_to_color_string(value: usize) -> String {
+    unimplemented!(
+        "convert the value {} into a string representation of color",
+        value
+    )
+}
+
+pub fn colors() -> Vec<ResistorColor> {
+    unimplemented!("return a list of all the colors in order")
```

Comment:
```suggestion
    unimplemented!("return a list of all the colors ordered by resistance")
```

Avoid confusion about i.e. alphabetical order.